### PR TITLE
Backend linter for PHP

### PR DESCRIPTION
**Pull request by alessandrobelli**
- [x] added backend linter using [Laravel Pint](https://laravel.com/docs/11.x/pint) 

it executes every push or pull request and address https://github.com/openqda/openqda/issues/5

@jankapunkt will it execute on my request or on merge?

### CODE REVIEW

**Comment by jankapunkt** [comment]
@alessandrobelli awesome! It already runs in this pr, see the checks section. Note that it's failing. If you want I can merge and you address the lint errors later or you can add fixes to this PR. To me both are okay

**Comment by alessandrobelli** [comment]
@jankapunkt changed it, the linter is pint, and we just need to type "pint" after we run composer install. 
Tests now are passing 🥳

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "require-dev": {
+        "laravel/pint": "^1.16"
+    }
+}
```
Comment (jankapunkt) [inline composer.json:5]: This is rather a question: do we need this separate `composer.json` for linting or could this be also placed into the existing composer file within the `web` directory?

**Review by jankapunkt — APPROVED** [review]

**Comment by jankapunkt** [comment]
@alessandrobelli please adress my question and resolve the conversation if you think it's not necessary, then I will merge it.
If you want to change it, please add the commit to this PR and reassign my review.

File: composer.json (hunk 1 above)
Comment (alessandrobelli) [reply composer.json:5]: I didn't realize that I created that, sorry! We need it only in the `web` directory, as you said.

**Review by jankapunkt — APPROVED** [review]

**Pull request merged**